src/main/java/org/fundacionjala/at15/katas/pokerhands/adrian/Flush.java

```diff
@@ -0,0 +1,33 @@
+package org.fundacionjala.at15.katas.pokerhands.adrian;
+
+public class Flush extends Hand {
+
+    public boolean theHandIsFlush(String hand) {
+        boolean result = false;
+        int counter = 0;
+        for (int ind = 0; ind < INDEX_FOUR; ind++) {
+            if (handSuit(hand)[ind] == handSuit(hand)[ind + 1]) {
+                counter++;
+            }
+        }
+        if (counter == INDEX_FOUR) {
+            result = true;
+        }
+        return result;
+    }
+
+    public String compareHandsWithFlush(String hand1, String hand2) {
```

Comment:
I liked your logic and they way that logic is distributed. One improvement I would suggest is to think inputs in terms of your own objects rather than strings.